examples/nffPktgen/generator/generator.go

```diff
@@ -229,6 +230,77 @@ func generateICMPIPv4(pkt *packet.Packet, config *PacketConfig, rnd *rand.Rand)
 	pktICMP.Cksum = packet.SwapBytesUint16(packet.CalculateIPv4ICMPChecksum(pktIP, pktICMP, pkt.Data))
 }
 
+func generatePcap(pkt *packet.Packet, config *PacketConfig, _ *rand.Rand) {
```

Comment:
The code to read the pcap file is derived from the read function in flow/flow.go.